src/System.Windows.Forms/src/System/Windows/Forms/ToolStripControlHost.StubSite.cs

```diff
@@ -40,27 +38,33 @@ public StubSite(IComponent control, IComponent host)
             /// <summary>
             ///  When implemented by a class, gets the container associated with the <see cref="ISite"/>.
             /// </summary>
-            IContainer ISite.Container => _owner.Site.Container;
+            IContainer? ISite.Container => _owner.Site?.Container;
 
             /// <summary>
             ///  When implemented by a class, determines whether the component is in design mode.
             /// </summary>
-            bool ISite.DesignMode => _owner.Site.DesignMode;
+            bool ISite.DesignMode => _owner.Site is null ? false : _owner.Site.DesignMode;
```

Comment:
Consider this way, like we used to do.
```suggestion
            bool ISite.DesignMode => _owner.Site?.DesignMode ?? false;
```